src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const COST_DAY = 40;
+  const DISCOUNT_OVER_3_DAYS = 20;
```

Comment:
don't use magic number inside variable name, what it will be 4 days in future
```suggestion
  const smallDiscount = 20;
```